lib/modules/hosts/imgur.js

```diff
@@ -38,20 +38,16 @@ modules['showImages'].siteModules['imgur'] = {
 				try {
 					var json = JSON.parse(response.responseText);
 					if (json.data.error || response.status === 404) {
-						// debugger;
 						def.reject(response, json.data.error);
 						return;
 					} else {
-						// debugger;
 						def.resolve(json.data);
 						return;
 					}
 				} catch (error) {
-					// debugger;
 					def.reject(error);
 					return;
 				}
-				// debugger;
 				def.reject(response);
```

Comment:
This function could do with even more cleanup: this line is unreachable, and the return statements are somewhat redundant.